ParquetClassLibrary/Sandbox/Parquets/AllParquets.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox.ID;
+
+namespace ParquetClassLibrary.Sandbox.Parquets
+{
+    /// <summary>
+    /// Stores all defined parquet models for use in sandbox-mode.
+    /// This collection is the source of truth about parquets for the rest of the library,
+    /// something like a color palette that other classes can paint with.
+    /// </summary>
+    public static class AllParquets
+    {
+        #region Test Values
```

Comment:
It feels a bit strange for the test values to be here, instead of closer to the test library, given the comment describing them as being specifically for QA Tests.

Reply:
Agreed, I will come up with a way to move them.